package.json

```
@@ -16,7 +16,7 @@
   ],
   "addon": true,
   "scripts": {
-    "test": "prettier . --check && bare test.js"
+    "test": "prettier . --check && bare test/index.js"
```

```suggestion
    "test": "prettier . --check && bare test.js"
```

Move `test/index.js` to `test.js` so we maintain the idiom that `test.js` is the single entry point for tests.

---

make sense 👍🏼